genvm/cmd/pricelist/main.go

```diff
@@ -11,20 +11,22 @@ type charge struct {
 }
 
 var (
-	chargeTXDATA   = charge{"txdata (per 8)", 128, "charged for storing transaction data"}
-	chargeSTORE    = charge{"store (per 8)", 5000, "charged for changing active state from zero to non-zero"}
-	chargeUPDATE   = charge{"update (per 8)", 725, "charged for updating active state"}
-	chargeLOAD     = charge{"load (per 8)", 182, "charged for loading active state"}
-	chargeEDVERIFY = charge{"edverify (per sig)", 3000, "charged for ed25519 verification"}
-	chargeSPAWN    = charge{"spawn", 30000, "charged for every spawn transaction"}
-	chargeTX       = charge{"tx", 20000, "charged for every transaction"}
+	chargeTXDATA        = charge{"txdata (per 8)", 128, "charged for storing transaction data"}
```

Comment:
Seems like this module should reuse constants from `genvm/core/gas` there, isn't it?

Reply:
this module is basically to help with design. i committed it so that it can reused by others if needed